src/index.ts

```diff
@@ -673,8 +685,182 @@ export interface DocumentContent {
   viewBackgroundFilter?: BackgroundFilter;
 }
 
+/**
+ * content metadata, stored with the "content" extension
+ *
+ * This largely contains description of how to render the document, rather than
+ * metadata about it.
+ */
+export interface TemplateContent<Constants extends string = BaseConstants> {
+  /** the template name */
+  name: string;
+  /** the template's author */
+  author: string;
+  /** Base64-encoded SVG icon image */
+  iconData: string;
+  /** category names this template belongs to (eg: "Planning", "Productivity") */
+  categories: string[];
+  /** labels associated with this template (eg: "Project management") */
+  labels: string[];
+  /** the orientation of this template */
+  orientation: 'portrait' | 'landscape';
+  /** semantic version for this template */
+  templateVersion: string;
+  /** template configuration format version (currently just `1`) */
+  formatVersion: number;
+  /**
+   * which screens the template supports:
+   *
+   * - `rm2`: reMarkable 2
+   * - `rmPP`: reMarkable Paper Pro
+   */
+  supportedScreens: ('rm2' | 'rmPP')[];
```

Comment:
One small extra comment. I don't think there are many still in use, but is it worth adding `rm` or `rm1` for original reMarkables? I ideally don't want this to error for those, although they can always load the raw json if that happens.

Reply:
I wasn't sure on this, because I don't know if rm/rm1 is the string and I don't know if that device/OS supports templates.

Reply:
yeah, that's fair. I'm cool with leaving it as it!